package.json

```diff
@@ -31,6 +31,8 @@
     "react-native": "zulip/react-native#master",
     "react-native-device-info": "^0.9.7",
     "react-native-drawer": "^2.3.0",
+    "react-native-menu": "^0.20.2",
+    "react-native-modalbox": "^1.3.9",
```

Comment:
Yeah! we can use built-in-modal. Working on this to make flow like this 
on option selected new page will be pushed containing the modal.  Right?